src/app/pages/ElectronicDelivery.jsx

```diff
@@ -237,35 +237,42 @@ class ElectronicDelivery extends React.Component {
             </div>
           )}
         </div>
-
-        <div>
-          {!_isEmpty(raiseError) && (
-            <div className='nypl-form-error' ref='nypl-form-error'>
-              <h2>Error</h2>
-              <p>
-                Please check the following required fields and resubmit your
-                request:
-              </p>
-              <ul>{this.getRaisedErrors(raiseError)}</ul>
-            </div>
-          )}
-          {!closedLocations.includes('') ? (
-            <ElectronicDeliveryForm
-              bibId={bibId}
-              itemId={itemId}
-              itemSource={this.state.itemSource}
-              submitRequest={this.submitRequest}
-              raiseError={this.raiseError}
-              error={error}
-              form={form}
-              defaultEmail={patronEmail}
-              searchKeywords={searchKeywords}
-              serverRedirect={serverRedirect}
-              fromUrl={this.fromUrl()}
-              onSiteEddEnabled={this.props.features.includes('on-site-edd')}
-            />
-          ) : null}
-        </div>
+        {!this.props.isEddRequestable ? (
```

Comment:
```suggestion
        {!item.eddRequestable ? (
```

You can get the item via two methods:

1. const item = bib.items.find((item) => item['@id'].includes(itemId));
2. const item = LibraryItem.getItem(bib, itemId);

Suggest to use option 1 becuase 2 will also `mapItem` which was already performed in the server request.